flake.nix

```diff
@@ -0,0 +1,55 @@
+{
+  description = "Python package for controlling an ElGato key light";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs";
+
+  outputs = { self, nixpkgs }:
+  let
+    pkgs = nixpkgs.legacyPackages.x86_64-linux;
+
+    leglight = pkgs.python38Packages.buildPythonPackage rec {
```

Comment:
This is opiniated and I have seen other flake snot follow this, but I try not to put too much logic like package creation in the flake.nix. I would put the package in a `default.nix` and this flake can just call that. I consider flakes more of a place where the sharing aspect comes in and including implementation logic hurts other's ability to see whats happening in your flake. `flake.nix` to me is a summary of your repo: what it needs, what it exports.